examples/polarify/pixi.lock

```diff
@@ -1,18 +1,19 @@
 version: 4
 environments:
-  blabliblublaplapliplupla:
```

Comment:
What happened here 😄

Reply:
Haha shit you found it. We're just discussing to remove unused environments from the lock file.